cfpb/consumerfinance.gov
# Paying for College - Fix form bugs

This PR will fix bugs with fields on the Make a Plan page. Fixes internal issues number 280 and 292.
.
## Changes
- Change variable names to address issues with the Make a Plan page.


## How to test this PR
1. Pull it down, get it running!
2. Check the follow fields on the Make a Plan page to make sure they match entries on earlier pages:
- Costs: Additional financial obligations
- Other loans: State loan fee %
- Other sources: estimated off-campus job income
- Other sources: estimated on-campus (not work study) job income
- Other sources: employer tuition reimbursement

## Checklist
